### PR TITLE
5.3.1 release notes

### PR DESCRIPTION
**Pull request by promanenko**

### CODE REVIEW

File: docs/modules/release-notes/pages/5-3-1.adoc [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+== Fixes
+
+* **SQL Browser:**
+** Fixed documentation link in SQL Browser. [MC-2183]
+** Fixed SQL Browser results scroll jump after the first 1000 rows. [MC-2182]
+** Fixed SQL Browser left panel message. [MC-2175]
+** Fixed SQL Browser column truncation. [MC-2106]
+* Fixed false-positive HealthCheck warnings during the cluster restart. [MC-1902]
+* Fixed various penetration testing findings. [MC-2061]
+* MC no longer allows running WAN Sync dry run if Merkle tree is not enabled. [MC-2145]
```
Comment (zhmaeff) [inline docs/modules/release-notes/pages/5-3-1.adoc:28]:
```suggestion
* MC no longer allows running WAN Sync check if Merkle tree is not enabled on the publisher. [MC-2145]
```

File: docs/modules/release-notes/pages/5-3-1.adoc [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+[[fixes-531]]
+== Fixes
+
+* **SQL Browser:**
+** Fixed documentation link in SQL Browser. [MC-2183]
+** Fixed SQL Browser results scroll jump after the first 1000 rows. [MC-2182]
+** Fixed SQL Browser left panel message. [MC-2175]
+** Fixed SQL Browser column truncation. [MC-2106]
+* Fixed false-positive HealthCheck warnings during the cluster restart. [MC-1902]
+* Fixed various penetration testing findings. [MC-2061]
```
Comment (Serdaro) [inline docs/modules/release-notes/pages/5-3-1.adoc:27]: Not sure if this entry should be included in this release notes document.

File: docs/modules/release-notes/pages/5-3-1.adoc [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+* MC no longer allows running WAN Sync dry run if Merkle tree is not enabled. [MC-2145]
+* MC no longer overrides WAN replication config params with default values. [MC-1461]
```
Comment (Serdaro) [inline docs/modules/release-notes/pages/5-3-1.adoc:29]: ... Replication configuration parameters with the ...

File: docs/modules/release-notes/pages/5-3-1.adoc [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+* Removed confusing info about user creation in logs. [MC-1766]
```
Comment (Serdaro) [inline docs/modules/release-notes/pages/5-3-1.adoc:30]: Removed inaccurate information related to the user creation in the logs. ...

File: docs/modules/release-notes/pages/5-3-1.adoc [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+* Fixed various Scripting-related issues. [MC-1543][MC-1530]
```
Comment (Serdaro) [inline docs/modules/release-notes/pages/5-3-1.adoc:31]: Fixed various scripting related issues including the saved scripts not being executed ...

**Review by Serdaro — APPROVED** [review]
Minor comments

File: docs/modules/release-notes/pages/5-3-1.adoc (hunk 2 above)
Comment (promanenko) [reply docs/modules/release-notes/pages/5-3-1.adoc:27]: ok, removed

**Pull request merged**